tests/test_lens.py

```diff
@@ -179,3 +215,26 @@ def test_empty_pipeline_run(
         training_data=classification_train_data,
     )
     my_pipeline.run()
+
+
+def test_lens_validation_no_sens_feat(init_lens_classification):
```

Comment:
This could be simplified like:

```python
@pytest.mark.xfail(raises=ValidationError)
def test_lens_validation_no_sens_feat(init_lens_classification):
    """
    Tests to ensure Lens will not allow running evaluators that require sensitive features without
    any sensitive features specified
    """
    lens, _, _ = init_lens_classification
    lens.assessment_data.sensitive_features = None
    lens.sens_feat_names = []
    lens.training_data = None
    lens.add(ModelFairness(["accuracy_score"])
```
There are also DataFairness, equity, and RankingFairness that have the same requirements. Can you parametrize if possible, or spawn new tests?